packages/apps/desktop-viewer-test/.env

```diff
@@ -0,0 +1,4 @@
+# ---- Configuration ----
+ITWIN_VIEWER_SCOPE ="email rbac-user:external-client projectwise-share urlps-third-party general-purpose-imodeljs-backend product-settings-service context-registry-service:read-only imodelhub organization profile openid imodeljs-router"
```

Comment:
would the desktop even need to use gpb? doesnt ht ebackend run locally, so you wouldnt need that scope, `general-purpose-imodeljs-backend` ?
also, what about the prdss one?

Reply:
there is still a "connected" mode in desktop mode where this will be used, but v2 with proper local briefcase support will likely warrant changes here

Reply:
Not sure I understand that. The desktop mode should still not need GPB right?

Reply:
currently it supports accessing models directly from the iTwin hub (similar to the desktop starter). Should this be removed?

Reply:
my mistake....this code actually isn't being called. Leftover from web copy I suppose. I'll remove

Reply:
removed